Library/Core/ColorResource.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import UIKit
```

Comment:
Maybe we can fix this by using `Double`, on 64 bit systems `CGFloat` is a `Double` so there it's a exact match. On 32 bit this will be a little inefficient since `CGFloat` there is a `Float`, but the `CGFloat` constructor will truncate the double value to a float and "fix" it that way.

Since nearly all systems we target are 64 bit I would be okay with that so we can remove the UIKit import again.